src/components/Confirm/index.js

```diff
@@ -172,12 +150,33 @@ class Confirm extends Component {
     }
   }
 
+  sendMultiframeSelfie = (snapshot, selfie, token, url, onSuccess, onError) => {
+    const snapshotData = {
+      file: {
+        blob: snapshot.blob,
+        filename: snapshot.filename
+      }
+    }
+    const { blob, filename, sdkMetadata } = selfie
+  
+    new Promise((resolve, reject) => {
+      uploadSnapshot(snapshotData, url, token, resolve, reject)
+    })
+    .then((res) => {
+      uploadLivePhoto({ file: { blob, filename }, sdkMetadata, snapshots: [res.id]}, url, token, onSuccess, onError)
```

Comment:
this line is perfect. But maybe for future readers, it is not clear what `res.id` is.

What if:

```
.then((res) => {
      const snapshotId = res.id
      uploadLivePhoto({ file: { blob, filename }, sdkMetadata, snapshots: [snapshot_id]}, url, token, onSuccess, onError)


```